src/wp-includes/Requests/Ipv6.php

```diff
@@ -90,7 +90,6 @@ public static function uncompress($ip) {
 	 * Example:  FF01:0:0:0:0:0:0:101   ->  FF01::101
 	 *           0:0:0:0:0:0:0:1        ->  ::1
 	 *
-	 * @see \WpOrg\Requests\IPv6::uncompress()
```

Comment:
```suggestion
	 * @see \WpOrg\Requests\IPv6::uncompress()
```

Whoops didn't mean to remove the original comment. Doh.